geo/geodataset.cpp

```diff
@@ -2203,8 +2203,13 @@ void GeoDataset::textureMesh(
                 }
             }
 
-            omesh.faces.emplace_back( indices[0], indices[1], indices[2]
-                                    , indices[0], indices[1], indices[2] );
+            omesh.faces.emplace_back(
```

Comment:
Wouldn't it be much easier (and nicer) to declare the `indices` array as:
```
geometry::Face::index_type indices[3]
```
?

Reply:
Sure, done, thanks.